hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncTableImpl.java

```diff
@@ -49,85 +51,103 @@
   private final ExecutorService pool;
 
   AsyncTableImpl(AsyncConnectionImpl conn, AsyncTable<AdvancedScanResultConsumer> rawTable,
-      ExecutorService pool) {
+    ExecutorService pool) {
     this.rawTable = rawTable;
     this.pool = pool;
   }
 
+  @WithSpan("AsyncTableImpl.getName")
   @Override
   public TableName getName() {
```

Comment:
Do we need to trace getName or getConfiguration methods? These methods should not cost too much time.

Reply:
OK, let's remove.